README.md

```diff
@@ -1,65 +1,75 @@
+> * [Enable Managed Service Identities](https://docs.microsoft.com/azure/active-directory/managed-service-identity/overview).
 > * Grant the required permissions for the web application to read data from Key vault.
 
-Before you proceed make sure that you are familiar with the [basic concepts](key-vault-whatis.md#basic-concepts).
+Before you proceed make sure that you are familiar with the [basic concepts](https://docs.microsoft.com/en-us/azure/key-vault/key-vault-overview).
 
 ## Prerequisites
 
-* [Node JS](https://nodejs.org/en/)
+* [Node.js](https://nodejs.org)
 * [Git](https://www.git-scm.com/)
 * [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest) 2.0.4 or later
 * An Azure subscription. If you don't have an Azure subscription, create a [free account](https://azure.microsoft.com/free/?WT.mc_id=A261C142F) before you begin.
 
-## Login to Azure
+## Log in to Azure
 
-To log in to Azure using the CLI, you can type:
+1. Open a command prompt, i.e. cmd, terminal, etc
+2. Execute the following command to log in to Azure
+`az login`
 
 ```azurecli
 az login
 ```
 
-## Create resource group
+## Create Resource Group
 
-Create a resource group with the [az group create](/cli/azure/group#az-group-create) command. An Azure resource group is a logical container into which Azure resources are deployed and managed.
+Create a Resource Group with the [az group create](https://docs.microsoft.com/en-us/azure/azure-resource-manager/manage-resources-cli) command. An Azure Resource Group is a logical container into which Azure resources are deployed and managed.
+
+When you create a Resource Group you have give it a unique custom name. Please think of a custom name for your resource group and replace the text below "YourResourceGroupName" with the custom name you created.
 
-Please select a Resource Group name and fill in the placeholder.
 The following example creates a resource group named *<YourResourceGroupName>* in the *eastus* location.
 
 ```azurecli
 # To list locations: az account list-locations --output table
-az group create --name "<YourResourceGroupName>" --location "East US"
+az group create --name "<YourResourceGroupName>" --location eastus
 ```
 
-The resource group you just created is used throughout this tutorial.
+The Resource Group you just created is used throughout this tutorial.
 
 ## Create an Azure Key Vault
 
-Next you create a Key Vault using the resource group created in the previous step. Although “ContosoKeyVault” is used as the name for the Key Vault throughout this article, you have to use a unique name. Provide the following information:
+Next you will create a Key Vault using the resource group created in the previous step. Although "ContosoKeyVault" is used as the name for the Key Vault throughout this article, you have to use a unique name. Provide the following information:
```

Comment:
Is ContosoKeyVault used?  I'm not seeing it referenced anywhere in this sample.

Reply:
removed